clickhouse/base/sslsocket.cpp

```diff
@@ -295,7 +295,7 @@ SSLSocketOutput::SSLSocketOutput(SSL *ssl)
 {}
 
 size_t SSLSocketOutput::DoWrite(const void* data, size_t len) {
-    return static_cast<size_t>(HANDLE_SSL_ERROR(ssl_, SSL_write(ssl_, data, len)));
+    return static_cast<size_t>(HANDLE_SSL_ERROR(ssl_, SSL_write(ssl_, data, static_cast<int>(len))));
```

Comment:
This is a very low probability scenario, but IMO we should explicitly check that piece of data is smaller than 2Gib here. Aaabd either send that data in 2Gib chunks or throw an exception here...

Reply:
This is all Greek to me,~~